Mod2/Lessons/Week1/ArtoftheGracefulExit.md

```diff
@@ -0,0 +1,76 @@
+# Art of the Graceful Exit
+
+## Learning Goals
+* Strategies for responding questions you can't answer.
+* Identify personal qualities you want to highlight.
+
+## Warm Up
+Imagine an interview where you are are asked a question you do not know the answer to.  
+* How do you feel? 
+* How do you answer?  
+* What impression does the interviewer have of you based on your answer?
+
+## Purpose of an Interview
+What is the job of an interviewer?
+
+Of course they are asking you some knowledge-based questions to see what you know and what you don’t know, but more importantly, they are trying to figure out what it would be like working with you.
+
+What type of person are you? What are you like working with teams? How much direction, supervision, and assistance will you need?
+
+When answering interview questions, you are providing the interviewer with lots of insight and information about the personal qualities you will bring to their team.
+
+### Demonstrating Personal Qualities
+
+Take a minute to jod down th personal qualities you would like the interviewer to walk away knowing about you.
+
+Examples:  
+* Curious  
+* Enthusiastic  
+* Humble  
+* Hard-working  
+* Growth mindset  
+* Good problem-solver  
+
+Keep these qualities in mind as you start to formulate you answers to questions you don't know the answer to.  Even if we can't give them the correct technical answer, we can still leave them with more knowledge about who we are as a developer, a teammate, and a human.
+
+**Some Example Responses**
+#### Clarifying Questions
+Interview question: “Can you give me some examples of how you might optimize an application?” Response: “By optimizing an application, do you mean making it faster? Could memoization or caching be an example of this?”
+
+What qualities is the interviewee displaying when they ask questions like this?
+
+#### Pivot
+Response: “I’m not familiar with that concept, but it reminds me of ____ . Could I talk a bit about that instead?”
+
+What qualities is this interviewee displaying?
+
+## Instructors Demonstrate Graceful (and Graceless) Interview Answers
+
```

Comment:
Do you have any instructor note guidance on how this part has gone before? Any examples of sample questions to ask, and/or things to focus on in the graceful and graceless would be helpful.